resources/lib/resumepoints.py

```diff
@@ -134,13 +133,32 @@ def update_online(self, asset_id, title, url, payload):
         """Update resumepoint online"""
         from json import dumps
         try:
-            get_url_json('https://video-user-data.vrt.be/resume_points/%s' % asset_id, headers=self.resumepoint_headers(url), data=dumps(payload).encode())
+            resumepoint_json = get_url_json('https://video-user-data.vrt.be/resume_points/%s' % asset_id,
+                                            headers=self.resumepoint_headers(url), data=dumps(payload).encode())
         except HTTPError as exc:
             log_error('Failed to (un)watch episode {title} at VRT NU ({error})', title=title, error=exc)
             notification(message=localize(30977, title=title))
             return False
+
+        # Update local representation and cache
+        self.update_local(asset_id, resumepoint_json)
```

Comment:
Hmm, I was confused here at first, because our internal representation should be identical to what we just pushed.

But I understand that pushing returns a complete representation of the resumepoints online. Just not entirely sure how this will work with concurrency.

Reply:
I'll remove this for now. Maybe this is something that can be useful in a next pull request.